wagtail/admin/api/actions/publish.py

```diff
@@ -0,0 +1,59 @@
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.shortcuts import get_object_or_404
+from rest_framework import fields, status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+
+from wagtail.api.v2.utils import BadRequestError
+from wagtail.core.actions.publish_page_revision import PublishPageRevisionAction
+
+from .base import APIAction
+
+
+class PublishPageAPIActionSerializer(Serializer):
+    changed = fields.BooleanField(default=True, required=False)
+    log_action = fields.BooleanField(default=True, required=False)
```

Comment:
I think we shouldn't allow changing the ``log_action`` on the admin API, since we don't currently have a way to validate that the provided action is a variant of a "publish" action. (a malicious user could pass "delete" here).